samples/tab-device-permissions/nodejs/src/components/GetGeoLocation.tsx

```diff
@@ -15,16 +16,24 @@ const GetGeoLocation = () => {
     microsoftTeams.app.initialize()
   })
 
-  // Method to get current user's geo location 
+  // Method to get current user's geo location
+  // If the value of allowChooseLocation is true, then the users can choose any location of their choice.
+  // If the value of allowChooseLocation is false, then the users cannot change their current location.
+  // If the value of showMap is false, the current location is fetched without displaying the map. 
+  // showMap is ignored if allowChooseLocation is set to true.
   function getLocation() {
+
     microsoftTeams.geoLocation.getCurrentLocation().then((location) => {
+      debugger;
       setGeoLocationValue(location)
     }).catch((error) => {
+      debugger;
```

Comment:
debugger should not be in PR, check and remove this

Reply:
resolved